doc/development.md

```diff
@@ -0,0 +1,127 @@
+# Development
+
+A Go 1.7+ environment is required. Docker and rkt (with SELinux set to Permissive) should be installed.
+
+## Binaries
+
+Develop the `update-operator` and `update-agent` apps locally.
+
+```
+make
+```
+
+Build  the `update-operator` and `update-agent` binaries in a build image with rkt.
+
+```
+make release-bin
+```
+
+## Container Image
+
+Build a container image.
+
+```
+make image
+```
+
+Push a development image to a personal image repository to test it on a cluster.
+
+```
+export IMAGE_REPO=quay.io/USERNAME/container-linux-update-operator
+make docker-push
```

Comment:
Yeah, I wanted either one to work. We can document just the Makefile flag version if you like